test/tests/functional/pbs_types.py

```diff
@@ -47,27 +47,23 @@ def test_pbs_size_deepcopy(self):
         """
         Test that deepcopy works for pbs.size type
         """
-        hook_content = ("import pbs\n"
-                        "import copy\n"
-                        "a = pbs.size(1000)\n"
-                        "b = copy.deepcopy(a)\n"
-                        "c = a\n"
-                        "pbs.logmsg(pbs.EVENT_DEBUG, "
-                        "'a=%s, b=%s, c=%s' % (a, b, c))\n"
-                        "d = pbs.size('1m')\n"
-                        "e = copy.deepcopy(d)\n"
-                        "f = d\n"
-                        "pbs.logmsg(pbs.EVENT_DEBUG, "
-                        "'d=%s, e=%s, f=%s' % (d, e, f))\n")
+        hook_content = ("""
+import pbs
+import copy
+a = pbs.size(1000)
+b = copy.deepcopy(a)
+c = a
+pbs.logmsg(pbs.EVENT_DEBUG, 'a=%s, b=%s, c=%s' % (a, b, c))
+d = pbs.size('1m')
+e = copy.deepcopy(d)
+f = d
+pbs.logmsg(pbs.EVENT_DEBUG, 'd=%s, e=%s, f=%s' % (d, e, f))
+""")
         hook_name = 'deepcopy'
-        hook_attr = {'enabled': 'true', 'event': 'execjob_begin'}
+        hook_attr = {'enabled': 'true', 'event': 'queuejob'}
         self.server.create_import_hook(hook_name, hook_attr, hook_content)
 
         j = Job(TEST_USER)
-        j.create_script('/bin/true')
         jid = self.server.submit(j)
```

Comment:
Please remove jid as it is unused... and codacy is complaining about it.

Reply:
Removed.